torch/_torch_docs.py

```diff
@@ -2312,9 +2312,11 @@ def merge_dicts(*dicts):
     inv = (u^T u)^{{-1}}
 
 Args:
-    input (Tensor): the input 2-D tensor :math:`u`, a upper or lower triangular
-           Cholesky factor
-    upper (bool, optional): whether to return a lower (default) or upper triangular matrix
+    input (Tensor): the input tensor :math:A of size :math:`(*, n, n)`,
```

Comment:
backticks on A missing.

Also, of -> or
For the docstring, how about making it "the input tensor :math:`A` of size :math:`(*, n, n)` consisting of symmetric positive-definite matrices where :math:`*` is zero or more batch dimensions." to make it clearer.

Reply:
The way we deal with batches in most of our linalg documentation is by adding a paragraph like this at the end of the "main" documentation:

> Supports input of float, double, cfloat and cdouble dtypes. Also supports batches of matrices, and if A is a batch of matrices then the output has the same batch dimensions.


See https://pytorch.org/docs/master/generated/torch.linalg.cholesky.html?highlight=cholesky#torch.linalg.cholesky for some ideas about how to structure the language

Reply:
Thanks for the feedback. Reflected all.